README_PRD.md

```diff
@@ -0,0 +1,233 @@
+# DOME-Marketplace GitOps
```

Comment:
Does this doc contain any information that cannot be included in the general README?

Reply:
Quite the reverse, the current README contains old AWS-based External DNS. That's why I wanted us to have a new readme, that is free of old stuff in order to avoid confusion. But if we manage to convince others to remove the DEMO instruction parts, I am fine with keeping all stuff in README.md

Reply:
Ok I see. But I would like your README to be the "official" README, so we might rename the current one as README_DEMO or just leave a note about AWS under the ExternalDNS section